CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 
 ### Changed
-
+- span.isRecording() now correctly returns false when not being sampled. (#1664)
```

Comment:
This will eventually reference the PR number instead of the issue number.  You can leave it as `TBD` for now.